tarodb/sqlc/queries/transfers.sql

```diff
@@ -127,3 +127,10 @@ WHERE asset_id = $1;
 -- name: DeleteSpendProofs :exec
 DELETE FROM transfer_proofs
 WHERE transfer_id = $1;
+
+-- name: InsertReceiverProofTransferAttempt :exec
+INSERT INTO receiver_proof_transfer_attempts (
```

Comment:
Ah ok, then I guess we'll do some aggregation to figure out how many times we tried, or when we need to back off.